tools/toolchain/scripts/stage8/install_spla.sh

```diff
@@ -189,10 +186,11 @@ export SPLA_HIP_LDFLAGS="${SPLA_HIP_LDFLAGS}"
 export CP_DFLAGS="\${CP_DFLAGS} IF_HIP(-D__OFFLOAD_GEMM|) IF_CUDA(-D__OFFLOAD_GEMM|) ${OFFLOAD_DFLAGS} IF_MPI(-D__SPLA|)"
 export CP_CFLAGS="\${CP_CFLAGS} ${SPLA_CFLAGS}"
 export SPLA_LIBRARY="-lspla"
-export SPLA_ROOT="$pkg_install_dir"
-export SPLA_INCLUDE_DIR="$pkg_install_dir/include/spla"
+export SPLA_ROOT="IF_HIP("${pkg_install_dir}-hip"|IF_CUDA("${pkg_install_dir}-cuda"|"${pkg_install_dir}"))"
+export SPLA_INCLUDE_DIR="${pkg_install_dir}/include/spla"
 export SPLA_VERSION=${spla-ver}
 export CP_LIBS="IF_MPI(${SPLA_LIBS}|) \${CP_LIBS}"
+export CP_LDFLAGS="\${CP_LDFLAGS} IF_HIP(${SPLA_HIP_LDFLAGS}|IF_CUDA(${SPLA_CUDA_LDFLAGS}|${SPLA_LDFLAGS}))"
```

Comment:
This line seems to confuse `parse_if.py`. I don't understand why, which is a bit worrying.

Nevertheless, is this line really need? It seems redundant with the following if/elif/else block.

Reply:
Thanks for spotting this. I forgot to remove it. I had issue with parenthesis put in the wrong place and I could figure out where it was coming from. This the typical if then else came handy. I will remove the line and leave the if-them-else structure.